### PR TITLE
Add Alias wallet how-tos

### LINKED ISSUE #416: Create example on how to use alias as wallet
## Description

We should create examples on how to use the Alias output (or the NFT output) as a wallet:

- [ ] Unlock Funds Owned by an Alias
- [ ] Transfer an NFT owned by an Alias

## Are you planning to do it yourself in a pull request?

Yes, someone from out team will work on it.

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added How-Tos on how to use an alias as wallet

Needs #618  to be merged.

## Links to any relevant issues

Fixes #416.

## Type of change

- Documentation Enhancement

## How the change has been tested

Tested locally (if possible)

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Clippy

File: bindings/nodejs/examples/how_tos/alias_wallet/request-funds.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        const aliasId = balance.aliases[0];
+        console.log(`Alias Id: ${aliasId}`);
+
+        // Get Alias address
+        const aliasAddress = Utils.aliasIdToBech32(aliasId, await (await wallet.getClient()).getBech32Hrp());
+        const faucetResponse = await (await wallet.getClient()).requestFundsFromFaucet(faucetUrl, aliasAddress);
+        console.log(faucetResponse);
+
+        await new Promise(resolve => setTimeout(resolve, 5000));
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias_wallet/request-funds.ts]: Often takes a bit longer, no?

File: documentation/sdk/docs/_admonitions/_alias_nft_wallet.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+:::tip Alias or NFT as wallet
+
+Although this How-To shows hot to use an Alias as wallet, the same code can be applied for NFTs with minor changes
```
Comment (Thoralf-M) [inline documentation/sdk/docs/_admonitions/_alias_nft_wallet.md]:
```suggestion
Although this How-To shows how to use an Alias as wallet, the same code can be applied for NFTs with minor changes
```

File: bindings/nodejs/examples/how_tos/alias_wallet/request-funds.ts (hunk 1 above)
Comment (Dr-Electron) [reply bindings/nodejs/examples/how_tos/alias_wallet/request-funds.ts]: Worked all the time for me 😆 .
But yeah guess I can make it 10s to be sure. I think I just copied it from somewhere (or maybe Copilot suggested it? :trollface: )

File: bindings/nodejs/examples/how_tos/alias_wallet/transaction.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            },
+        ];
+        const options = {
+            mandatoryInputs: [input],
+            allowMicroAmount: false,
+        };
+        const transaction = await account.sendAmount(params, options);
+        await account.retryTransactionUntilIncluded(transaction.transactionId);
+        console.log(
+            `Transaction with custom input: https://explorer.iota.org/testnet/transaction/${transaction.transactionId}`,
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias_wallet/transaction.ts:76]: But it's provided in mandatoryInputs :trollface: There is also customInputs which will not automatically select other outputs than the provided ones

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/examples/how_tos/alias_wallet/transaction.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Wallet, initLogger, Utils } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/simple_transaction/simple-transaction.ts
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/how_tos/alias_wallet/transaction.ts]: No

Comment (thibault-martinez) [inline bindings/nodejs/examples/how_tos/alias_wallet/transaction.ts]:
```suggestion
// yarn run-example ./how_tos/alias_wallet/transaction.ts
```

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**